### PR TITLE
Add SPM Support

### PR DESCRIPTION
**Pull request by dejanskledar**
- Added SPM support
- Fixed missing UIKit imports
- Updated the Example project

✅  Tested through SPM and CocoaPods.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            targets: ["Sentinel"]
+        ),
+        .library(
+            name: "SentinelLoggie",
+            targets: ["SentinelLoggie"]
+        ),
+        .library(
+            name: "SentinelCollar",
+            targets: ["SentinelCollar"]
+        )
```
Comment (jabou) [inline Package.swift:23]: Will this pull Loggie and Collar automatically or will the developer be able to select which tools he wants to install?

**Review by jabou — APPROVED** [review]

File: Package.swift (hunk 1 above)
Comment (dejanskledar) [reply Package.swift:23]: No it wont pull automatically, the developer will be able to select which tool he wants to include.

Comment (dejanskledar) [reply Package.swift:23]: image

Comment (jabou) [reply Package.swift:23]: great! <3

**Pull request merged**